runtime/compiler/runtime/codertinit.cpp

```diff
@@ -216,6 +217,13 @@ J9JITConfig * codert_onload(J9JavaVM * javaVM)
          goto _abort;
       }
 
+   /* initialize classChainValidationMutex */
+   if (!classChainValidationMutex)
+      {
+      if (!(classChainValidationMutex = TR::Monitor::create("JIT-ClassChainValidationMutex")))
```

Comment:
I forgot when `codert_onload` is executed, but it would be nice to create this monitor only if needed, thus maybe we should move this inside `initClassChainValidationBitVectors()`

Reply:
Yeah I was thinking about whether I can avoid creating the monitor if I don't need it. I'll look into it.

Reply:
I changed where the monitor is created; essentially the monitor is only needed after we init the map, so I'll change the code to create the monitor when we go to init the map.